core/ledger/pvtdatastorage/reconcile_missing_pvtdata.go

```diff
@@ -21,91 +22,169 @@ import (
 //     from the above created data entries
 // (2) create a db update batch from the update entries
 // (3) commit the update batch to the pvtStore
-func (s *Store) CommitPvtDataOfOldBlocks(blocksPvtData map[uint64][]*ledger.TxPvtData) error {
+func (s *Store) CommitPvtDataOfOldBlocks(
+	blocksPvtData map[uint64][]*ledger.TxPvtData,
+	deprioritizedMissingData ledger.MissingPvtDataInfo,
```

Comment:
reconciler will pass `unreconciledData` to the `kvledger` and the `kvledger` would also forward the parameter here. I made it this way so that the only `kvledger` knows this logic. I can change this to `unreconciledData` if you don't object.